From 25359686722682da36f583ab2c5da451b4486e2c Mon Sep 17 00:00:00 2001
From: Aswin Murugan <aswin.murugan@oss.qualcomm.com>
Date: Mon, 5 Oct 2026 11:25:06 +0530
Subject: [PATCH] configs: shikra: enable OP-TEE support in U-Boot

Enable CONFIG_TEE, CONFIG_OPTEE and CONFIG_OPTEE_LIB in
Shikra U-Boot defconfig to support OP-TEE integration.

Upstream-Status: Pending
Signed-off-by: Aswin Murugan <aswin.murugan@oss.qualcomm.com>
---
 configs/qcom_shikra_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configs/qcom_shikra_defconfig b/configs/qcom_shikra_defconfig
index 6decfb55bd72..ae830cd635f4 100644
--- a/configs/qcom_shikra_defconfig
+++ b/configs/qcom_shikra_defconfig
@@ -28,3 +28,7 @@ CONFIG_ENV_MMC_USE_SW_PARTITION=y
 CONFIG_ENV_MMC_SW_PARTITION="logfs"
 CONFIG_ENV_OFFSET=0x0
 CONFIG_ENV_SIZE=0x4000
+
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+CONFIG_OPTEE_LIB=y
